test/gtest/ucp/test_ucp_tag_mem_type.cc

```diff
@@ -94,6 +94,17 @@ class test_ucp_tag_mem_type: public test_ucp_tag {
         mem_buffer::pattern_check(rbuf, length, 2, r_mem_type);
     }
 
+    size_t max_test_length(unsigned exp) const
+    {
+        size_t max_length = static_cast<size_t>(pow(10.0, exp));
+        return ucs_max(10, max_length % ucs::test_time_multiplier());
```

Comment:
i think better reduce num of iterations. to not lose coverage

Reply:
but here we don't have iterations, we go through `i=[1..7]` and calculate message size as `pow(10, i)`

Reply:
iterate 1..3 and calculate size pow(10,i*2)?

Reply:
maybe do not change it at all, as the benefit does not look significant enough

Reply:
we do all available pairs of host,cuda,cuda-managed,rocm,rocm-managed, i.e. 5^5=125

Reply:
> iterate 1..3 and calculate size pow(10,i*2)?

done